### PR TITLE
Some small fixes to make it work.

### PR DESCRIPTION
**Pull request by mvanbaak**
First of all, thanks for the work. It almost works perfectly.
While testing it I found some small issues, which I fixed.
- Amazon released 2016.03.3 AMI so use that as default
- /etc/redhat-release is not needed and confuses ansible
- /vagrant cannot be mounted without Guest additions and breaks. So don't mount it
- The base AMI misses gcc and kernel-devel packages. They are needed to compile the kernel module. Install them.

### CODE REVIEW

**Comment by jjpersch** [comment]
Thanks for the contribution! 
I'll followup after testing.

File: scripts/export-vmdk.sh [hunk 1]
```diff
@@ -82,11 +85,6 @@ EOF
 # Disable sudo tty requirement
 sed -i.bak -e 's/^Defaults    requiretty/#Defaults    requiretty/g' $MNT/etc/sudoers
 
-# Ensure vagrant is able to detect guest as redhat
-# https://github.com/mitchellh/vagrant/blob/7caaffbf389fb298840eafb1edcdf554b977182e/plugins/guests/redhat/guest.rb#L7
-touch $MNT/etc/redhat-release
```
Comment (jjpersch) [inline scripts/export-vmdk.sh]: This was added for Vagrantfiles that attempted to configure private networking.
Will need to verify if still required.

Example:
```
n.vm.guest = :redhat
n.vm.network "private_network", ip: 172.30.0.10
```

File: scripts/install-guest-additions.sh [hunk 2]
```diff
@@ -22,9 +22,10 @@ export VAGRANT_CWD="$(mktemp -d -t "${name}")"
 cat >"${VAGRANT_CWD}/Vagrantfile" <<EOF
 Vagrant.configure(2) do |config|
   config.vm.box = "${name}"
+  config.vm.synced_folder ".", "/vagrant", disabled: true
   config.ssh.insert_key = false
   config.vm.provision :shell,
-    inline: "yum -y update --security"
+    inline: "yum -y update --security && yum -y install gcc kernel-devel"
```
Comment (jjpersch) [inline scripts/install-guest-additions.sh]: Merged as https://github.com/monetate/ectou-export/pull/8

File: export.py [hunk 3]
```diff
@@ -184,15 +184,15 @@ def get_parser():
...
+                   default="amzn-ami-hvm-2016.03.3.x86_64-gp2",
                    help="Source image name")
 
     g = parser.add_argument_group("Builder")
     g.add_argument("--builder-ami-owner",
                    default="amazon",
                    help="Builder image owner")
     g.add_argument("--builder-ami-name",
-                   default="amzn-ami-hvm-2016.03.1.x86_64-gp2",
+                   default="amzn-ami-hvm-2016.03.3.x86_64-gp2",
```
Comment (jjpersch) [inline export.py]: Merged as https://github.com/monetate/ectou-export/pull/7

**Comment by mvanbaak** [comment]
👍

File: scripts/install-guest-additions.sh [hunk 4]
```diff
@@ -27,6 +27,7 @@ vagrant box add --name "${name}" "${box}"
 cat >"${VAGRANT_CWD}/Vagrantfile" <<EOF
 Vagrant.configure(2) do |config|
   config.vm.box = "${name}"
+  config.vm.synced_folder ".", "/vagrant", disabled: true
```
Comment (jjpersch) [inline scripts/install-guest-additions.sh]: This is on line 36 now and can be removed.

Comment (mvanbaak) [inline scripts/install-guest-additions.sh]: done.

**Comment by jjpersch** [comment]
Merged update to latest linux AMI in #10

**Pull request closed without merging**